lib/grpnames.g

```diff
@@ -25,8 +25,8 @@ NAMES_OF_SMALL_GROUPS :=
   [ "C5 : C4", "C20", "C5 : C4", "D20", "C10 x C2" ], [ "C7 : C3", "C21" ],
   [ "D22", "C22" ], [ "C23" ],
   [ "C3 : C8", "C24", "SL(2,3)", "C3 : Q8", "C4 x S3", "D24",
-      "C2 x (C3 : C4)", "(C6 x C2) : C2", "C12 x C2", "C3 x D8", "C3 x Q8",
-      "S4", "C2 x A4", "C2 x C2 x S3", "C6 x C2 x C2" ], [ "C25", "C5 x C5" ],
+      "C2 x (C3 : C4)", "C3 : D8", "C12 x C2", "C3 x D8", "C3 x Q8", "S4",
+      "C2 x A4", "C2 x C2 x S3", "C6 x C2 x C2" ], [ "C25", "C5 x C5" ],
   [ "D26", "C26" ],
   [ "C27", "C9 x C3", "(C3 x C3) : C3", "C9 : C3", "C3 x C3 x C3" ],
   [ "C7 : C4", "C28", "D28", "C14 x C2" ], [ "C29" ],
```

Comment:
Why did you do that checkout? Anyway, it makes little sense to me. What you want instead is to again checkout your local branch `SemidirectDecompositions`. That's the result of your rebase. It now diverse from `origin/SemidirectDecompositions`, i.e. what is on the server - that's normal and to be expected, as you performed a rebase. The larger number 303 of additional commits in the local branch `SemidirectDecompositions` stems from the fact that it now also includes all the work that happened on the master branch since april...
So, this work should now be pushed to your github clone of gap, which will then be reflected in this PR.

This should do it:
```
git checkout SemidirectDecompositions
git push --force
```